leetcode/src/79.c

```diff
@@ -0,0 +1,60 @@
+int getPointKey(int i, int j, int boardSize, int boardColSize){
+    return boardSize * boardColSize * i + j;
+}
+
+const int directionsSize = 4;
+const int directions[4][2] = {{-1, 0}, {1, 0}, {0, -1}, {0, 1}};

```

Comment:
C is funny sometimes. I have never seen `const` vector of vector of `int` using `int` 😂 I guess it doesn't matter but the type is not `int` here.

Reply:
That would be called a matrix, I think. 😃